src/window/basic_frame.rs

```diff
@@ -1,8 +1,6 @@
 use std::io::{BufWriter, Seek, SeekFrom, Write};
 use std::sync::{Arc, Mutex};
 
-use byteorder::{NativeEndian, WriteBytesExt};
```

Comment:
Is the `byteorder` crate used elsewhere? If not it should be removed from the dependencies.

Reply:
if they are only used in the examples, it can be downgraded to a dev-dependency then.